RunGrepTest

```diff
@@ -859,8 +859,13 @@ if [ $? -ne 0 ]; then
   echo "RC=2" >>testtrygrep
 else
 
-  (cd $srcdir; unset LC_ALL; LC_CTYPE=locale.bad $valgrind $vjs $pcre2grep abc /dev/null) >>testtrygrep 2>&1
+  (cd $srcdir; unset LC_ALL; export LC_CTYPE=locale.bad; $valgrind $vjs $pcre2grep abc /dev/null >>$builddir/testtrygrep 2>&1) >testtemp1grep 2>&1
```

Comment:
This is likely broken in at least Solaris.

POSIX shell requires export and setting the variables in two different commands

Reply:
Of course, thank you for the reminder of that pitfall.